service/src/main/java/com/softserve/borda/services/impl/BoardListServiceImpl.java

```diff
@@ -38,14 +36,14 @@ public BoardList deleteTicketFromBoardList(BoardList boardList, Ticket ticket) {
             ticketService.deleteTicketById(ticket.getId());
             return boardListRepository.save(boardList);
         } catch (Exception e) {
-            throw new CustomFailedToDeleteEntityException(e.getMessage());
+            throw new com.softserve.borda.exceptions.CustomFailedToDeleteEntityException(e.getMessage());
```

Comment:
Can you explain a reason of adding relative path to CustomFailedToDeleteEntityException class?

Reply:
Idea must have imported it this way after dividing project into modules and I missed it. Will fix it